docs/reference/feature-gates.md

```diff
@@ -61,6 +61,7 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `true` | Beta| 1.10 | |
 | `MountContainers` | `false` | Alpha | 1.9 | |
 | `MountPropagation` | `false` | Alpha | 1.8 | |
```

Comment:
nit ... line 63 should read

```
 | `MountPropagation` | `false` | Alpha | 1.8 | 1.9 |
```

This clearly indicates that the Alpha stage ends at 1.9 version.